From bba00811a2b8f874f6bc146675de3850bbe9ec8f Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.4-bba00811a2b-20210928T125228Z-avarab@gmail.com>
In-Reply-To: <cover-0.4-00000000000-20210928T125228Z-avarab@gmail.com>
References: <cover-0.4-00000000000-20210928T125228Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 27 Sep 2021 01:24:02 +0200
Subject: [PATCH 1/4] builtin.h: remove cmd_tar_tree() declaration
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The cmd_tar_tree() function itself was removed in
925ceccf050 (tar-tree: remove deprecated command, 2013-11-10).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/builtin.h b/builtin.h
index 16ecd5586f0..8a58743ed63 100644
--- a/builtin.h
+++ b/builtin.h
@@ -225,7 +225,6 @@ int cmd_submodule__helper(int argc, const char **argv, const char *prefix);
 int cmd_switch(int argc, const char **argv, const char *prefix);
 int cmd_symbolic_ref(int argc, const char **argv, const char *prefix);
 int cmd_tag(int argc, const char **argv, const char *prefix);
-int cmd_tar_tree(int argc, const char **argv, const char *prefix);
 int cmd_unpack_file(int argc, const char **argv, const char *prefix);
 int cmd_unpack_objects(int argc, const char **argv, const char *prefix);
 int cmd_update_index(int argc, const char **argv, const char *prefix);
-- 
2.33.0.1340.ge9f77250f2b

